indico/modules/events/surveys/templates/manage_questionnaire.html

```diff
@@ -1,35 +1,90 @@
-                                data-title="{% trans %}Edit question{% endtrans %}"
-                                class="js-question-dialog">
-                        <strong>{{ question.title }}</strong>
-                    </a>
+        <div class="sortblock i-box-content" style="width: 100%">
+            <ul class="group-list no-content-before">
+                {% for question in survey.questions %}
+                <li style="cursor: move">
+                    <table class="dragHandle" style="table-layout: fixed;width: 100%">
+                        <tbody>
+                            <tr>
+                                <td><strong>{{ question.title }}</strong></td>
+                                <td>{{ question.description }}</td>
+                                <td>{{ form_row(preview_form[question.title], orientation='horizontal',
+                                                widget_attrs=preview_widget_attrs) }}</td>
+                                <td>
+                                    <a data-href="{{ url_for('.remove_question', question) }}" class="i-button icon-remove js-remove-question right"></a>
+                                    <a href="#" data-href="{{ url_for('.edit_question', question) }}"
+                                                data-title="{% trans %}Edit question{% endtrans %}"
+                                                data-question-id="{{ question.id }}"
+                                                class="i-button js-question-dialog icon-edit right"></a>
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
                 </li>
-            {% else %}
-                <li>{% trans %}No questions defined{% endtrans %}</li>
-            {% endfor %}
+                {% endfor %}
             </ul>
         </div>
     </div>
+    <script>
+        $('.js-remove-question').on('click', function(e) {
+            e.preventDefault();
+            var $this = $(this);
+            var msg = $T('Do you really want to remove this question room from the survey?');
+            new ConfirmPopup($T('Delete this question ?'), msg, function(confirmed) {
+                if (!confirmed) {
+                    return;
+                }
+
+                $('<form>', {
+                    action: $this.data('href'),
+                    method: 'post'
+                }).appendTo('body').submit();
+            }).open();
+        });
+
+        $('#questions').tablesorter({
+            sortables: '.sortblock ul',
+            sortableElements: '> li',
+            placeholderElement: '<li></li>',
+            handle: '.dragHandle',
+            onReorder: function(sortable) {
+                var questionIdsInOrder = [];
+                $('[data-question-id]').each(function(index, obj) {
+                    questionIdsInOrder.push($(obj).data('question-id'));
+                });
+
+                $.ajax({
```

Comment:
There should at least be `error: handleAjaxError`. I would also add this to make it clear to the user that changing the order results in something being saved:

    complete: IndicoUI.Dialogs.Util.progress()